diff --git a/Android.mk b/Android.mk
index b3d7e62..f6d2d92 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,6 +1,7 @@
 ###################################
 # Build the libchromium_net library
 
+ifneq ($(PRODUCT_PREBUILT_WEBVIEWCHROMIUM),yes)
 LOCAL_PATH := $(call my-dir)
 include external/chromium/third_party/libevent/Android.mk
 include external/chromium/third_party/modp_b64/Android.mk
@@ -492,3 +493,4 @@ ifneq ($(strip $(WITH_ADDRESS_SANITIZER)),)
 endif
 
 include $(BUILD_SHARED_LIBRARY)
+endif
